packages/terra-form-select/src/single/Frame.jsx

```diff
@@ -256,21 +267,15 @@ class Frame extends React.Component {
     // event.relatedTarget returns null in IE 10 / IE 11
     if (event.relatedTarget == null) {
       // IE 11 sets document.activeElement to the next focused element before the blur event is called
-      if (document.querySelector(this.selectMenu) === document.activeElement) {
-        return;
-      }
-    // Modern browsers support event.relatedTarget
-    } else if (document.querySelector(this.selectMenu) === event.relatedTarget) {
-      return;
-    }
-
-
-    this.setState({ isFocused: false });
-
-    this.closeDropdown();
 
-    if (this.props.onBlur) {
-      this.props.onBlur(event);
+      setTimeout(() => {
+        if (document.querySelector(this.selectMenu) !== document.activeElement) {
+          _onBlur();
+        }
+      }, 10);
```

Comment:
Why is this timeout needed?

Reply:
There is [this strange behavior](https://github.com/cerner/terra-core/issues/2648#issuecomment-539909574) I noticed w.r.t. the blur event on IE that seemed to align with our [existing inline code comment](https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/single/Frame.jsx#L258). When I read these comments [[1]](https://github.com/facebook/react/issues/3751#issuecomment-415863466) [[2]](https://github.com/catamphetamine/react-responsive-ui/blob/77b7ebc2e43b35595b06cda3ac697b75505e2174/source/utility/focus.js#L30-L45) about blur behaves on IE in React, that seemed to make sense and their trick worked. 

Why 10? I honestly randomly guessed it.

Reply:
You are right. This unusual. If this is the work-around, we should add an inline comment with links to these issues so those less familiar with this code will understand why this timeout is here.

Reply:
Done.